keyboards/kumo/config.h

```diff
@@ -0,0 +1,30 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+#include "config_common.h"
+#define VENDOR_ID       0xFEAE
+#define PRODUCT_ID      0x8847
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    TheVan Keyboards
```

Comment:
We should probably put this in thevankeyboards directory

Reply:
Ahh I did not know this existed, thanks!